README.md

```diff
@@ -13,6 +14,30 @@ A suite of Nagios style checks and metrics covering the basic needs for monitori
 * [Service](https://github.com/ncr-devops-platform/nagios-foundation/blob/master/cmd/check_service/README.md)
 * [User and Group](https://github.com/ncr-devops-platform/nagios-foundation/blob/master/cmd/check_user_group/README.md)
 
+## Using
+Use this collection of applications as [Sensu Go Checks](https://docs.sensu.io/sensu-go/5.5/reference/checks/) in your Sensu deployment. For example, to check every 60 seconds that the signage application is running on a remote kiosk where the Sensu Agent is subscribed to `signage`, run:
+
+```
+$ cat << EOF | sensuctl create
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "process_signage",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check_process --name signage_app",
+    "interval": 60,
+    "publish": true,
+    "subscriptions": [
+      "signage"
+    ]
+  }
+}
+EOF
```

Comment:
It might be a good idea to add an asset dependency config section with a valid url and sha512 sum for one of our releases

Reply:
Added with 2nd commit.